ctrl_z/backup.py

```diff
@@ -57,20 +57,31 @@ def restore(
 
         logger.info("Finished restore of %s", self.base_dir)
 
-    def create_directories(self):
+    def create_directories(self, version=None):
         logger.debug("Checking/creating folder tree for backups")
 
         paths = (self.db_dir, self.files_dir)
+
+        if version:
```

Comment:
Why not add self.version_path to paths so it is created like the other directores?

Reply:
While that works locally, for some reason it did not work when executing the backup through a pod.